wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
Oh, the message buffer needs to be reset!
That was not needed before as the buffer was always cleaned by CoolProp when writing error message, but here is needed to force this 0 after the last character.
I correct this.